src/index.js

```diff
@@ -838,7 +821,11 @@ const members = {
   },
 };
 
-if (!('attachShadow' in document.createElement('div'))) {
+if (shadowDomV1) {
+  // then we should probably not be loading this
+} else if (shadowDomV0) {
+  v0();
+} else {
   const commProto = Comment.prototype;
```

Comment:
Eventually I'd like to do the same thing here that I did with `v0()` to simplify this file.